### PR TITLE
MVP-2604: 'subscribeAlert' method in 'EvntTypeWalletBalanceRow'

### PR DESCRIPTION
**Pull request by eric-notifi**
Implement 'subscribeAlert/ unsubscrigbeAlert' in EventTypeWalletBalanceRow that allowed client to subscribe alert using either hooks or frontendClient

### CODE REVIEW

File: packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx [hunk 1]
```diff
@@ -52,6 +70,45 @@ export const EventTypeWalletBalanceRow: React.FC<
     didFetch.current = true;
   }, [alertName, alerts]);
 
+  const subscribeAlert = useCallback(
+    async (
+      alertDetail: Readonly<{
+        eventType: EventTypeItem;
+        inputs: Record<string, unknown>;
```
Comment (juni-notifi) [inline packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx:77]: is this always `{}`?

File: packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx [hunk 2]
```diff
@@ -52,6 +70,45 @@ export const EventTypeWalletBalanceRow: React.FC<
...
+      }>,
+    ): Promise<SubscriptionData> => {
+      if (isCanaryActive) {
+        return subscribeAlertByFrontendClient(frontendClient, alertDetail);
```
Comment (juni-notifi) [inline packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx:81]: how does this method receive the `connectedWallets`?

**Review by juni-notifi — COMMENTED** [review]
Hopefully quick questions

File: packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx (hunk 2 above)
Comment (eric-notifi) [reply packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx:81]: Inside `subscribeAlertByFfrontendClient`, the `ensureAlert` --> `ensure` --> `ensureSourceAndFilters` are called. Then all the connectedWallets source will be fetched. hope it is not a wrong understanding?

File: packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx (hunk 1 above)
Comment (eric-notifi) [reply packages/notifi-react-card/lib/components/subscription/EventTypeWalletBalanceRow.tsx:77]: oh.. yeah, this `inputs` field is always empty object. The reason why it exists is for the interface consistency of [subscribeAlertByFrontendClient](https://github.com/notifi-network/notifi-sdk-ts/blob/b6881792f87ffecb8713ce83aa4a23eaecb2853f/packages/notifi-react-card/lib/utils/frontendClient.ts#L10). Hope it make sense.

**Review by juni-notifi — APPROVED** [review]

**Review by juni-notifi — APPROVED** [review]

**Pull request merged**